lib/pages/pool/MyLiqudity/bloc/MyLiquidityBloc.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:ax_dapp/pages/pool/MyLiqudity/models/MyLiquidityItemInfo.dart';
+import 'package:ax_dapp/repositories/usecases/GetAllLiquidityInfoUseCase.dart';
+import 'package:ax_dapp/service/Controller/usecases/GetWalletAddressUseCase.dart';
+import 'package:ax_dapp/util/BlocStatus.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:equatable/equatable.dart';
+
+part 'MyLiquidityEvent.dart';
+part 'MyLiquidityState.dart';
+
+class MyLiquidityBloc extends Bloc<MyLiquidityEvent, MyLiquidityState> {
+  final GetWalletAddressUseCase controller;
+  final GetAllLiquidityInfoUseCase repo;
+
+  MyLiquidityBloc({required this.repo, required this.controller})
+      : super(MyLiquidityState.initial()) {
+    on<LoadEvent>(_mapLoadEventToState);
+  }
+
+  Future<void> _mapLoadEventToState(
+      LoadEvent event, Emitter<MyLiquidityState> emit) async {
+    emit(state.copyWith(status: BlocStatus.loading));
+    final String? walletAddress = controller.getWalletAddress();
+    try {
+      if (walletAddress != null) {
+        final response =
+            await repo.fetchAllLiquidityTokensInfoForWallet(walletAddress: walletAddress);
+        final isSuccess = response.isLeft();
+        if (isSuccess) {
+          final myLiquidityItemInfos =
```

Comment:
`liquidityDetailItemList` or `liquiditityInfoItemList`... information does not have a dimension of countability so it sounds awkward to say infos... 

also aren't these `pools`? Instead of items cant we say `LiquidityPoolDetailsList`?

Reply:
Yeah, `LiquidityPoolDetailsList` sounds good.